features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Apis/Details/Forum.js

```diff
@@ -1,20 +1,72 @@
 import  React from 'react'
 import Paper from 'material-ui/Paper';
 import Grid from 'material-ui/Grid';
-import Typography from 'material-ui/Typography';
+import Button from 'material-ui/Button';
+import Card from 'material-ui/Card';
+import {Row} from 'antd';
+
 
 class Forum extends React.Component{
     constructor(props){
         super(props);
+        this.state = {
+            commentList: [],
+            comment: '',
+        };
+        this.updateCommentString = this.updateCommentString.bind(this);
+        this.handleAddComment = this.handleAddComment.bind(this);
+
     }
+
+    updateCommentString(event) {
+        this.setState({comment: event.target.value});
+    }
+
+    handleAddComment() {
+        var api = new Api();
+        let commentInfo = {"commentText": this.state.comment};
+        let promise = api.addComment(this.api_uuid, commentInfo);
+        promise.then(
+            response => {
+                this.handleGetAllComments();
+                this.setState({comment: ''});
+                message.success("Comment added successfully");
+            }).catch(
+            error => {
+                message.error("Error occurred while adding comments!");
+            }
+        );
+    }
+
     render(){
+        const {commentList, comment} = this.state;
         return (
             <Paper>
                 <Grid container className="tab-grid" spacing={0} >
                     <Grid item xs={12}>
-                        <Typography type="display1" gutterBottom >
-                            <span style={{fontSize: "50%"}}>Comments page</span>
-                        </Typography>
+                        <div>
+                            <p>Comments</p>
+                            <Row>
+                            <textarea cols="180" rows="4" value={comment}
+                                      onChange={this.updateCommentString}> </textarea>
+                            </Row>
```

Comment:
Shall we try to get rid of this Ant base code

Reply:
Sure , will do it

Reply:
Thanks @dakshika , Fixed with https://github.com/wso2/carbon-apimgt/pull/4982/commits/705b0caa544635802a2edc4b710e50af0adc57d5 commit